src/Elasticsearch.Net/Transport/Sniff/SniffResponse.cs

```diff
@@ -7,39 +7,38 @@ namespace Elasticsearch.Net
 {
 	internal class SniffResponse
```

Comment:
maybe make this class public with an internal constructor so that only `Elasticsearch.Net` can instantiate it?

That way `AddressRe` can be public and renamed to `AddressRegex`